code/controllers/subsystem/processing/greyscale.dm

```diff
@@ -43,6 +43,16 @@ PROCESSING_SUBSYSTEM_DEF(greyscale)
 		CRASH("Invalid colors were given to `GetColoredIconByType()`: [colors]")
 	return configurations[type].Generate(colors)
 
+/datum/controller/subsystem/processing/greyscale/proc/GetColoredIconByType_universal_icon(type, list/colors, target_icon_state)
```

Comment:
bro wtf is this proc name

Reply:
i bow to our new naming standard, PascalSnakeReverseCamelCase.